pkg/cloudprovider/provider/anexia/provider.go

```diff
@@ -248,6 +203,8 @@ func provisionVM(ctx context.Context, client anxclient.Client) error {
 	return updateMachineStatus(reconcileContext.Machine, *status, reconcileContext.ProviderData.Update)
 }
 
+var _engsup3404mutex sync.Mutex
```

Comment:
we should probably give this another name, given how people outside Anexia cannot see this ticket and this is going upstream